bin/lib/aws/translation-api-limit.js

```diff
@@ -1,8 +1,9 @@
 const AWS = require('aws-sdk');
+const Utils = require('../utils/utils');
 
-const limitTable = process.env.LIMIT_TABLE;
-const region = process.env.AWS_REGION;
-const apiLimits = JSON.parse(process.env.API_CHAR_LIMITS);
+const limitTable = Utils.processEnv('LIMIT_TABLE');
+const region     = Utils.processEnv('AWS_REGION', {validateString: /eu/});
```

Comment:
validateString statement works, but I don't think we should be using it for this env var.

Reply:
yep

Reply:
done